web/admin/src/request/types.ts

```diff
@@ -207,6 +207,12 @@ export interface DomainAnalysisConfluenceResp {
   title?: string;
 }
 
+export interface DomainAnydocUploadResp {
```

Comment:
在 types.ts 文件中，添加了新的类型定义，如 DomainAnydocUploadResp、V1FeishuGetDocReq 等。这些新的类型定义增强了类型安全性。

```suggestion
// 添加新的类型定义以增强类型安全性
export interface DomainAnydocUploadResp {
  code?: number;
  data?: string;
  err?: string;
}

// 添加 Feishu 相关的类型定义
export interface V1FeishuGetDocReq {
  doc_id: string;
  file_type?: string;
  id: string;
  kb_id: string;
  space_id?: string;
}
// ... 其他新增的类型定义